functions_and_scope_assignment.js

```diff
@@ -0,0 +1,283 @@
+//QUESTION 1
+
+/*Write a function that returns the largest of two numbers.
+If they are equal, return the second one.*/
+//
+// console.log('--------Question 1---------')
+// const findTheLargeNum = (num1, num2) => {
+//   if (num1 > num2) {
+//     console.log(`${num1} is greater than ${num2}`);
+//     return true;
```

Comment:
We want number to be returned depending on which is bigger. In this case we don't need to return true but rather num1 or num2.